ArduPlane/sensors.cpp

```diff
@@ -77,6 +81,45 @@ void Plane::read_airspeed(void)
     if (ahrs.airspeed_estimate(&aspeed)) {
         smoothed_airspeed = smoothed_airspeed * 0.8f + aspeed * 0.2f;
     }
+
+    float wind_max = ahrs.get_max_wind();
+    if(ahrs.airspeed_sensor_enabled() && airspeed.use() == 3) {
```

Comment:
this logic should be split out into a separate function, maybe check_airspeed_health()

Reply:
Done and moved to the airspeed library